### DISCUSSION TITLE
Status of version 1.3

### DISCUSSION THREAD

**Discussion by tacman** [Spomky-Labs/pwa-bundle#250] [General]
What's the status of version 1.3?  In particular, do you have a gut feeling for when it'll be released?

There's been discussion about a stimulus controller, but the documentation page at https://pwa.spomky-labs.com/ doesn't yet reflect 1.3.  As I recall, there are some differences in how the cache is defined in the pwa.yaml  file.  

I have a related bundle that integrates dexie (a wrapper for IndexedDB) and twigjs, and I want to put together a demo program to show how those bundles work, and would like to use pwa-bundle 1.3, the stimulus controller, etc.

**Comment by Spomky** [comment]
Hello @tacman,

I am working on a complete revision of the demo. It helps my tracking bugs or misconceptions of the new stimulus components.
The demo is not complete at the moment. Please refer to https://github.com/Spomky-Labs/phpwa-demo/issues/42 to know what is already implemented.

Also, the goal of this demo is to demonstrate everything can be done  without a single line of JS.
The version 1.3 is not mature enough for now.

**Comment by tacman** [comment]
> The version 1.3 is not mature enough for now.

Okay, but the demo is at 1.3.x.  As is https://github.com/Spomky/qotd  Are there any demos that show off what 1.2 can do?  I'm inclined to use version 3 and just copy pwa.yaml in my own app, but better would probably be to do that with 1.2.

In particular, I'm struggling with the Install button, which is one of my favorite features, to only show the install button if the app isn't installed, and ideally show a launch button if the app is installed but running in a web page.

Of course, installing the app to run offline requires that the page cache be configured.   And I think that cache configuration changed, so I go to demo apps for guidance.

I have some demo apps that are mentioned in the show-and-tell section.  I've been updating them to Symfony 7.2, and would love to update them to the latest pwa-bundle.  I guess I can post specific issues here with them.

**Reply by tacman** [reply]
Actually, the "install" button for QOTD does work as expected, when the app is installed the button is not visible.

**Comment by tacman** [comment]
ping...

It's been kinda quiet here.  Any word on 1.3 (with documentation and demos)?  A lot seems to work, could you simply release what works and mark everything else (in the documentation) as experimental?  1.3 is markedly better than 1.2.

Having the features that exist now, running with the latest Symfony and PHP, documented with a demo would be great.

**Reply by back-2-95** [reply]
I see changes incoming to 1.3 branch, see https://github.com/Spomky-Labs/pwa-bundle/compare/1.2.5...1.3.x

But there are a lot of changes, which mean lots of testing etc. Dunno how much will be included yet.
